src/config/constants.ts

```diff
@@ -1,5 +1,5 @@
 const constants = {
-  APP_NAME: '' // TODO: just a template, remove it later
+  secret_key: 'SECRET_KEY' || '' // TODO: just a template, remove it later
```

Comment:
It is a great idea to put it here, but It should be in CAPS to write a well-structured code to follow the same convention